.buildkite/cache-builder.yml

```diff
@@ -14,6 +14,7 @@ common_params:
   # Common environment values to use with the `env` key.
   - &common_env
     IMAGE_ID: xcode-13
+    FASTLANE_HIDE_PLUGINS_TABLE: 1 # Don't print noisy plugins table and save logs lines
```

Comment:
Is it because that would set the env var on the host machine as opposed to inside the macOS VM?

That makes me wonder if there's an official way for us to declare  env vars within the VM… from the pipeline, like you tried to here? The only options to do it that I know of would be:
 - declare it in the `buildkite-hooks/environment` like you did… but that then declares it on all buildkite projects then (regardless of the project, agent, and pipeline). Which was fine for this one but might not always be what we need
 - Declare it in mobile-secrets, by exporting it in the `env` file associated with the Buildkite project
 - Export it inside the `.sh` script that we invoke on each command of each step needing that env var

So in the end there's no key in the pipeline.yml that would allow us to declare an env var inside the agent VM, and we have to resort to one of those 3 options instead?